tests/unit_tests/test_ice.py

```diff
@@ -0,0 +1,71 @@
+from server.ice_servers.coturn import CoturnHMAC
+from unittest import mock
+from twilio.rest import Client as TwilioRestClient
+from server.ice_servers.nts import TwilioNTS
+import pytest
+
+
+@pytest.fixture
+def coturn_hmac(coturn_hosts, coturn_keys):
+    return CoturnHMAC(coturn_hosts=coturn_hosts, coturn_keys=coturn_keys)
+
+
+@pytest.fixture
+def twilio(twilio_sid, twilio_token):
+    with mock.patch('twilio.rest.Client', mock.Mock(spec=TwilioRestClient)):
+        mocked = TwilioNTS(sid=twilio_sid, token=twilio_token)
+        mocked.client.tokens.create = mock.Mock(
+            return_value=FakeTwilioServers()
+        )
+        return mocked
+
+
+class FakeTwilioServers:
+    def __init__(self):
+        self.ice_servers = \
+            dict(

```

Comment:
Why are you using `dict`? IMO using `{"urls": ["a", "b", "c"], ...}` is easier to read. (You can still format it nicely)